advancedblacklist/core.py

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) 2021 - Jojo#7791
+# Licensed under MIT
+
+import logging
+from contextlib import suppress
+from typing import Optional, Set
+
+import discord  # type:ignore
+from redbot.core import Config, commands
+from redbot.core.bot import Red
+from redbot.core.utils.chat_formatting import humanize_list
+
+from .abc import CompositeMetaclass
+from .commands import Blacklist, Whitelist
+from .commands.utils import (add_to_blacklist, add_to_whitelist, clear_blacklist, clear_whitelist,
+                             in_blacklist, in_whitelist, remove_from_blacklist,
+                             remove_from_whitelist)
+from .const import __authors__, __version__, _config_structure
+from .patch import destroy, init
+
+log = logging.getLogger("red.jojocogs.advancedblacklist")
+
+
+def api_tool(ctx: commands.Context):
+    from .commands.utils import api
+
+    return api
```

Comment:
```suggestion
def api_tool(ctx: commands.Context):
    from .commands.utils import api

    return api
```
This... seems interesting to me. Context isn't used and the function isn't returning anything other than an file, couldn't this be done instead?

```py
from .command.utils import api as api_tool
```

If you decide to go with this, you'd want to remove the instances where the function is called, seeing as its no longer callable.

Reply:
This function is used for the dev env variable, which takes a context object. This is designed so I can return the api file for debugging purposes